### PR TITLE
Td 05

### PR DESCRIPTION
**Pull request by IsraelGboluwaga**

### CODE REVIEW

File: .idea/workspace.xml [hunk 1]
```diff
@@ -1,12 +1,16 @@
 <?xml version="1.0" encoding="UTF-8"?>
```
Comment (samuelkayode) [inline .idea/workspace.xml:1]: I thin it should hav started with a comment line for me to really know what you are doing.

File: views/login.hbs [hunk 2]
```diff
@@ -2,10 +2,10 @@
 
     <div class="container">
         <div class="col-sm-10 col-sm-offset-1">
-            <h1 class="display-4"> {{ header }} </h1>
+            <h1 class="display-4" id="header"> {{ header }} </h1>
             <br>
```
Comment (samuelkayode) [inline views/login.hbs:7]: This doesnt really matter though, but why the break after the br tag? A comment would have been better

File: public/javascripts/scripts.js [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+    // backDelay: 5000,
+    // loop
+    // loop: false,
+    // false = infinite
+    // loopCount: 5,
+    // show cursor
+    showCursor: false,
+    // character for cursor
+    cursorChar: "|",
+    // attribute to type (null == text)
```
Comment (samuelkayode) [inline public/javascripts/scripts.js:19]: A comment line should be easier to understand. thats why its a comment. Lay man, might not know==

**Review by samuelkayode — CHANGES_REQUESTED** [review]

File: public/javascripts/scripts.js (hunk 3 above)
Comment (IsraelGboluwaga) [reply public/javascripts/scripts.js:19]: Ok sir

File: views/login.hbs (hunk 2 above)
Comment (IsraelGboluwaga) [reply views/login.hbs:7]: It's for easier readability.

**Pull request merged**